employee-rostering-distribution/src/main/assembly/assembly-optaweb-employee-rostering-sources.xml

```diff
@@ -6,6 +6,7 @@
   <formats>
     <format>dir</format>
     <format>zip</format>
+    <format>jar</format>
```

Comment:
@Christopher-Chianelli Now we have both `-sources.jar` and `-sources.zip` and they're almost identical (the JAR has `META-INF/MANIFEST.MF` in addition). I think we should remove the ZIP in the next PR.